src/Worker.Extensions.DurableTask/TaskOrchestrationContextExtensionMethods.cs

```diff
@@ -14,20 +19,65 @@ namespace Microsoft.DurableTask;
 /// </summary>
 public static class TaskOrchestrationContextExtensionMethods
 {
+    private const int DefaultPollingIntervalMilliseconds = 30000;
+    private const string PollingInterval = "df.http.defaultAsyncRequestSleepTimeMilliseconds";
+
     /// <summary>
     /// Makes an HTTP call using the information in the DurableHttpRequest.
     /// </summary>
     /// <param name="context">The task orchestration context.</param>
     /// <param name="request">The DurableHttpRequest used to make the HTTP call.</param>
     /// <returns>DurableHttpResponse</returns>
-    public static Task<DurableHttpResponse> CallHttpAsync(this TaskOrchestrationContext context, DurableHttpRequest request)
+    public static async Task<DurableHttpResponse> CallHttpAsync(this TaskOrchestrationContext context, DurableHttpRequest request)
     {
         if (context is null)
         {
             throw new ArgumentNullException(nameof(context));
         }
```

Comment:
It's important to ensure that all logger names have a prefix so that customers can easily configure the visibility across multiple related loggers. I looked at the .NET Isolated worker repo and found out that they have a convention for this (see [here](https://github.com/Azure/azure-functions-dotnet-worker/blob/ab45bacd09effcc40d046d1451451e74f50d1019/extensions/Worker.Extensions.Http.AspNetCore/src/Infrastructure/ExtensionTrace.cs#L15)). So I suggest we name this something like `Microsoft.Azure.Functions.Worker.Extensions.DurableTask.CallHttp`.